src/app/handle-page/edit-handle-page/edit-handle-page.component.html

```diff
@@ -6,6 +6,13 @@
              [(ngModel)]="handle"
              [placeholder]="'handle-table.edit-handle.form-handle-input-placeholder' | translate">
     </div>
+    <div class="form-group">
+      <label for="handleId">ID</label>
```

Comment:
Please add it to `*.json5` messages. I know the ID will be the same in every language, but for consistency we should use those messages.

Reply:
ok